src/main/java/microsoft/exchange/webservices/data/core/EwsUtilities.java

```diff
@@ -1212,11 +1113,10 @@ public static void validateParamAllowNull(Object param, String paramName)
    * @param paramName Name of the param.
    * @throws Exception the exception
    */
-  public static void validateParam(Object param, String paramName)
-      throws Exception {
-    boolean isValid = false;
+  public static void validateParam(Object param, String paramName) throws Exception {
+    boolean isValid;
 
-    if (param != null && param instanceof String) {
+    if (param instanceof String) {
```

Comment:
Actually `param != null && param instanceof String` is equivalent to `param instanceof String`, because `null instanceof String == false`. I just simplified this statement.